index.bs

```diff
@@ -1329,23 +1340,39 @@ Removing AB or TAG members</h5>
 	Individual participants of the [=Advisory Board=] or the [=Technical Architecture Group=]
 	can be <dfn export lt="remove|removal">removed</dfn> from those groups
 	if they are found by their peers
-	to be grossly neglecting their duties,
-	or to be acting in a way that seriously hampers the group's ability to function.
+	to be grossly neglecting their duties
+	or acting in a way that seriously hampers the group's ability to function;
+	or if they are violating policies or other standards of behavior
+	(such as applicable laws or the <a href="https://www.w3.org/policies/code-of-conduct/#Reporting">Code of Conduct</a>)
+	in a way that significantly impacts the safety or integrity
+	of W3C or its community.
+
+	Removal <em class=rfc2119>should</em> be a last resort,
+	after reasonable attempts at remediation have failed.
 
-	The [=AB=] or [=TAG=] <em class=rfc2119>must</em> hold a hearing
+	The [=AB=] or [=TAG=] <em class=rfc2119>must</em> hold an <i>in camera</i> hearing
 	on the potential [=removal=] of a participant
-	if requested by at least three of the participants in the group.
+	if requested by at least three of the participants in the group
+	or if requested by a [=Team decision=]
```

Comment:
I'm not sure that we have a need to have the Team do this.  There's a Team liaison who can find three TAG/AB members to initiate this process.

Reply:
That seems unnecessarily indirect... for things like Code of Conduct violations or repeated unchecked violations of the confidentiality policy, I think it makes more sense to allow the Team to request directly. They can make their case once, to everyone simultaneously, without needing to develop side-channel discussions.

Regarding potential for abuse, I think it's highly unlikely the Team would try to DDOS the AB or the TAG using this ability, and the proceedings are private so if the Team uses it for frivolous reasons the group can easily contain the problem.

Reply:
I'm operating on a different principle, which is that the W3C is a member-led organization.  That principle does not specially privilege the Team.

Reply:
That principle does not.... but W3C is a collaboration between the Membership and the Team, and we depend on the Team for certain functions: mediating conflicts, acting as an intermediary, and managing disciplinary action are some of them. In that role, it makes sense for the Team to be able to request a hearing directly.

In addition, I think it's healthier for the Team to be able to approach the TAG or AB on such matters directly, collectively, and forthrightly.

Reply:
This PR has been broken up into pieces. Discussion on this aspect to be continued at #1052